github-management/kubernetes-repositories.md

```diff
@@ -136,6 +139,41 @@ ecosystem. Carries the endorsement of the Kubernetes community.
      Kubernetes steering committee elections. in the Kubernetes community
    * Repository must be approved by SIG-Architecture
 
+## Maintenance Mode
+
+In order to ensure that our repositories are as up to date & relevant as possible,
+pruning is an essential activity. Prior to the clean-up, however, active dependencies 
+need to be addressed. To ease the transition from Active to Archived, the Kubernetes
+community has introduced **Maintenance Mode** following precedents from the below
+projects:
+
+- [CNCF Project Archiving Process v1.0](https://github.com/cncf/toc/blob/main/process/archiving.md)
+- [The Apache Attic](https://attic.apache.org/)
+
+### Roadmap
+
+When in maintenance mode, the community can expect the following from the owners
+of the project/repository:
+
+  * No concrete plan on introduction of new features
+  * Incoming PR(s) will not be looked at in any regular cadence
+  * Issues being filed will end up being ignored
+  * Minimal upkeep for golang updates, dependency updates
+  * Secuurity-related features/updates to be taken care of
+  * Explicit guidance around removal/transition to active state
+
+### How-tos
+
+  * SIG Chairs or TLs can open PRs to add the label and a preamble in the README.md
+ for the project.
+  * Steering committee liaisons can recommend a project/repository to transition to
+ maintenance mode during the annual reporting process.
+  * When in doubt, SIG Architecture will be the decision maker.
```

Comment:
These look more like "rules" than the process itself. Could we split this into "rules" vs "process" like in the other sections?